src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -440,7 +452,11 @@ void ResumptionDataProcessor::ProcessHMIRequests(
   }
 
   for (const auto& it : requests) {
-    ProcessHMIRequest(it, true);  // subscribe_on_response = true
+    const bool is_request =
+        application_manager::MessageType::kRequest ==
+        (*it)[strings::params][strings::message_type].asInt();
+
+    ProcessHMIRequest(it, is_request);
```

Comment:
`ProcessHMIRequest(it, is_request = false) ` is possible condition? Confusing

Reply:
@LuxoftAKutsan yes, as in some cases  `ResumptionDataProcessor` is sending not only requests, but notifications as well

Reply:
@LuxoftAKutsan I renamed all these function to `ProcessMessageToHMI` to avoid confusions. As well as input parameter, which is `message` but not `request`
See https://github.com/LuxoftSDL/sdl_core/pull/70/commits/6b705d58c4fe3bd504d40281528a7ff5a3e03858